### ISSUE TITLE
Virtual host creation UI

### ISSUE THREAD

**Issue by mmativ** [leokhoa/laragon#153]
Hi, is Laragon has a plan to create a Virtual Host creation UI like WAMP?

**Comment by leokhoa** [comment]
@mmativ: No, I don't think it is needed as Virtual Host in Laragon is automatic which is more advanced than WAMP.

**Comment by mmativ** [comment]
@leokhoa I know there is a way to manually edit the vhost but for some instance, i need to create 5 vhost that point on same directories.

**Comment by ClausMunch** [comment]
I have had the same need several times.
Would be nice to be able to do it from the GUI.

**Comment by JABirchallHR** [comment]
Laragon supports nginx and apache, if they made a GUI for apache they would need to create one for nginx too. 
There also isnt a need for it, it would be best for you to instead learn how to configure apche/nginx. Its a more valuable skill.
